tests/data/update-pickles.py

```diff
@@ -1,13 +1,15 @@
 import pickle
 
 import cobra.test
+import optlang
 
 from cameo import load_model
 
-ijo = load_model('iJO1366.xml')
+ijo = load_model('iJO1366.xml', solver_interface=optlang.glpk_interface)
 with open('iJO1366.pickle', 'wb') as out:
-    pickle.dump(ijo, out)
+    pickle.dump(ijo, out, protocol=2)
```

Comment:
I prefer using `pickle.HIGHEST_PROTOCOL` although that has equalled two for the past decade or so.